integration-tests/utils/integration-utils.js

```diff
@@ -182,15 +182,18 @@ const delay = ms => {
 }
 
 const dismissNotifications = async page => {
-  await delay(3000) // delayed for any alerts to show
-  const notificationCloseButtons = await page.$x(
-    '(//a | //*[@role="button"] | //button)[contains(@aria-label,"Dismiss") and not(contains(@aria-label,"error"))]')
-
+  const closeButtonXpath = '(//a | //*[@role="button"] | //button)[contains(@aria-label,"Dismiss") and not(contains(@aria-label,"error"))]'
+  try {
+    await page.waitForXPath(closeButtonXpath, { timeout: 3000 }) // 3 seconds wait for any alerts to show
```

Comment:
Again, this seems like it would work ok if there were only one notification. However, if multiple notifications appear with a slight delay between them, then it looks like this would see the first notification, close it, and then continue, leaving the second notification blocking parts of the UI.